channeldb/kvdb/etcd/readwrite_bucket_test.go

```diff
@@ -403,3 +403,60 @@ func TestBucketSequence(t *testing.T) {
 
 	require.Nil(t, err)
 }
+
+func TestKeyClash(t *testing.T) {
```

Comment:
nit: missing godoc, function body is also a little light on comments

Reply:
done, ptal